server/cluster/cluster.go

```diff
@@ -53,9 +54,16 @@ import (
 
 var backgroundJobInterval = 10 * time.Second
 
+type storeProgressStatus int
+
 const (
-	clientTimeout              = 3 * time.Second
-	defaultChangedRegionsLimit = 10000
+	clientTimeout                                  = 3 * time.Second
+	defaultChangedRegionsLimit                     = 10000
+	progressMetricsTimer                           = 5    // progress run interval second
+	progressMaxRetryLimit                          = 20   // max check times
+	progressFinish             storeProgressStatus = iota // progress progress finish
+	progressStoreOffLine                                  // store offline
```

Comment:
How about `progressNoTarget`?

Reply:
progress no target is only arm to  up,  how to describe offline